etc/compile/vscode_compile_task.sh

```diff
@@ -1,25 +1,17 @@
 #!/usr/bin/env bash
 set -e
 
-REPO_DIR=$(git rev-parse --show-toplevel)
-COMPILE_DIR=$(dirname "${BASH_SOURCE[0]}")
-GO_VERSION=$(cat "${COMPILE_DIR}/GO_VERSION")
-VERSION_ADDITIONAL=-$(git log --pretty=format:%H | head -n 1)
-LD_FLAGS="-X github.com/pachyderm/pachyderm/src/client/version.AdditionalVersion=${VERSION_ADDITIONAL}"
-
-DOCKER_OPTS=()
-DOCKER_OPTS+=("--build-arg=GO_VERSION=${GO_VERSION}")
-DOCKER_OPTS+=("--build-arg=LD_FLAGS=${LD_FLAGS}")
-DOCKER_OPTS+=("--build-arg=GC_FLAGS=${GC_FLAGS}")
-DOCKER_OPTS+=("--memory=3gb")
-
 if [[ "${OS}" == "Windows_NT" ]]; then
   # On windows, we need to copy over the environment variables for connecting to docker
   eval "$(minikube docker-env --shell bash)"
 fi
 
-DOCKER_BUILDKIT=1 docker build "${DOCKER_OPTS[@]}" --progress plain -f Dockerfile.pachd -t pachyderm_build "${REPO_DIR}"
-docker build "${DOCKER_OPTS[@]}" -t pachyderm/pachd "${REPO_DIR}/etc/pachd"
-docker tag pachyderm/pachd pachyderm/pachd:local
-docker build "${DOCKER_OPTS[@]}" -t pachyderm/worker "${REPO_DIR}/etc/worker"
-docker tag pachyderm/worker pachyderm/worker:local
\ No newline at end of file
+REPO_DIR=$(git rev-parse --show-toplevel)
+COMPILE_DIR=$(dirname "${BASH_SOURCE[0]}")
+VERSION_ADDITIONAL=-$(git log --pretty=format:%H | head -n 1)
+
+export DOCKER_BUILDKIT=1
+export GOVERSION=$(cat "${COMPILE_DIR}/GO_VERSION")
+export LD_FLAGS="-X github.com/pachyderm/pachyderm/src/client/version.AdditionalVersion=${VERSION_ADDITIONAL}"
+
+goreleaser release -p 1 --snapshot --skip-publish --rm-dist -f "${REPO_DIR}/goreleaser/docker.yml"
```

Comment:
I am guessing there is a good reason why we cannot call make docker-build here? If you could, this code would look like...
cd $REPO_DIR
make docker-build

Reply:
So I wrote this to require as few extra tools as possible, and `make` doesn't exist on windows without jumping through a few hoops - unfortunately I think what you said is just the right thing to do, so I rewrote this to use `make` and we can get rid of all this duplication.